recipes/nvcloth/1.1.6/conanfile.py

```diff
@@ -170,6 +180,10 @@ def package_info(self):
         else:
             self.cpp_info.libs = ["NvCloth{}".format(debug_suffix)]
 
-        if not self.options.shared:
+        self.cpp_info.includedirs = ['include']
+        self.cpp_info.libdirs = ['lib']
+        self.cpp_info.bindirs = ['bin']
+
```

Comment:
```suggestion
```
These are configured already by default: https://docs.conan.io/2/reference/conanfile/methods/package_info.html#cpp-info-library-and-build-information